.github/workflows/ci.yml

```diff
@@ -11,6 +11,7 @@ env:
   CVC4_VERSION: "4.1.8"
   YICES_VERSION: "2.6.2"
   SAW_VERSION: "0.8"
+  LLVM_CONFIG: "llvm-config-12"
```

Comment:
SAW 0.8 only supports LLVM 10 and earlier. I'd recommend pinning LLVM to version 10.

Reply:
Is it better to pin LLVM to 10 or bump SAW to the new release (0.9)?

Reply:
We could do the latter, but it would require some further reshuffling to accommodate the new SAW version. The CI [shows one example of this](https://github.com/GaloisInc/saw-demos/runs/4218712498?check_suite_focus=true): because SAW 0.9 no longer links in ABC, any proofs involving the `abc` tactic will fail unless an ABC executable is on the PATH. This would be doable with [`what4-solvers`](https://github.com/GaloisInc/what4-solvers), but it would probably require enough changes to the CI that it would be best to leave that for a separate PR.